ome/training-notebooks
# Version upgrade

bump omero, r-gateway and cellprofiler version

To test:
* go to https://ome-lochy.openmicroscopy.org
* select snoopycrimecop/training-notebooks:master_merge_daily
* check that you can connect to merge-ci
* Check that CP notebook works
* Check that R notebook works
cc @pwalczysko @dominikl
